hasher-matcher-actioner/src/OpenMediaMatch/blueprints/curation.py

```diff
@@ -89,6 +93,10 @@ def bank_update(bank_name: str):
         storage.bank_update(bank, rename_from=rename_from)
     except ValueError as e:
         abort(400, *e.args)
+    except IntegrityError:
+        abort(403, "Bank already exists")
+    except Exception as e:
+        abort(500, *e.args)
```

Comment:
blocking: What happens if you don't catch the exception at all? If you call this abort I think it may "hide" the root exception (by replacement). 

I suspect the default behavior is also to 500, so you may not need this explicit abort.